src/getCachedValue.ts

```diff
@@ -57,7 +57,6 @@ export async function getCachedValue<Value>(
         report({ name: 'refreshValueStart' });
         void cachified({
           ...context,
-          reporter: () => () => {},
```

Comment:
This existed to silence reporting for background refreshes. Not needed anymore since reporter isn't part of context anymore.